lisa/tools/systemctl.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+from lisa.util.process import ExecutableResult
+
+
+class Systemctl(Tool):
```

Comment:
Merge systemctl and service into a virtual tool. It can base on different distro to use one of them. It will be easier to use.